rbi/stdlib/psych.rbi

```diff
@@ -497,36 +497,1032 @@ module Psych
   # Load the document contained in +filename+.  Returns the yaml contained in
   # +filename+ as a Ruby object, or if the file is empty, it returns
   # the specified +fallback+ return value, which defaults to +false+.
-  sig { params(filename: String, fallback: T.untyped).returns(T.untyped) }
+  sig { params(filename: T.any(String, Pathname), fallback: T.untyped).returns(T.untyped) }
   def self.load_file(filename, fallback: false); end
 end
 
+class Psych::Exception < RuntimeError
+end
+
+class Psych::BadAlias < Psych::Exception
+end
+
+class Psych::ClassLoader
+  BIG_DECIMAL = ::T.unsafe(nil)
+  CACHE = ::T.unsafe(nil)
+  COMPLEX = ::T.unsafe(nil)
+  DATE = ::T.unsafe(nil)
+  DATE_TIME = ::T.unsafe(nil)
+  EXCEPTION = ::T.unsafe(nil)
+  OBJECT = ::T.unsafe(nil)
+  PSYCH_OMAP = ::T.unsafe(nil)
+  PSYCH_SET = ::T.unsafe(nil)
+  RANGE = ::T.unsafe(nil)
+  RATIONAL = ::T.unsafe(nil)
+  REGEXP = ::T.unsafe(nil)
+  STRUCT = ::T.unsafe(nil)
+  SYMBOL = ::T.unsafe(nil)
+
+
+  def big_decimal(); end
+
+
```

Comment:
Small suggestion to run a quick subst over this to remove the double newlines

Reply:
TIL `cat -s`